.github/workflows/backup-20250905-114531/ci-with-codecov.yml

```
@@ -0,0 +1,72 @@
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v4.1.0
+        with:
+          version: 10.0.0
+          run_install: false
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)